tests/test_actions.py

```diff
@@ -424,7 +451,8 @@ def test_raises_exception_for_too_few_tests(self):
     @mock.patch("screenpy.actions.see_all_of.See")
     def test_calls_see_for_each_test(self, MockedSee, Tester):
         num_tests = 3
-        tests = ((mock.Mock(), mock.Mock()),) * num_tests
+        tests = ((mock.create_autospec(Question, instance=True),
+                  mock.create_autospec(BaseResolution, instance=True)),) * num_tests
```

Comment:
I think this block would be a little clearer using a comprehension instead. Also i'm pretty sure the resulting tuple using this method would have 3 references to the same question/resolution pair. 

Maybe something like this:

```python
tests = (
    (
        mock.create_autospec(Question, instance=True),
        mock.create_autospec(BaseResolution, instance=True)
    )
    for test in range(num_tests)
)
```